src/rpdk/core/project.py

```diff
@@ -222,14 +225,21 @@ def write_settings(self):
             raise InternalError("Internal error (Plugin returned invalid runtime)")
 
         def _write(f):
+            executable_entrypoint_dict = (
+                {"executableEntrypoint": self.executable_entrypoint}
+                if self.executable_entrypoint
+                else {}
+            )
             json.dump(
                 {
                     "typeName": self.type_name,
                     "language": self.language,
                     "runtime": self.runtime,
                     "entrypoint": self.entrypoint,
                     "testEntrypoint": self.test_entrypoint,
+                    "executableEntrypoint": self.executable_entrypoint,
```

Comment:
what's the difference between adding this key here and dumping the contents of the other dict?

Reply:
Good catch. I should not have been adding it. Updated unit tests to catch this case